secrets/pangeo-181919.json

```diff
@@ -0,0 +1,30 @@
+{
+	"type": "ENC[AES256_GCM,data:mJFhTAgDkzazdN3XqSZk,iv:ZAc39oA/yaK4Fjll2113xkuP1JazR7p4zMw5NDyJa7M=,tag:IpjlDkcrQ7oxq8iNALMo8A==,type:str]",
+	"project_id": "ENC[AES256_GCM,data:muHRDdmA0GGj+6QL/A==,iv:BfTAtG/gBo4tSo4nB/tGnp6mfVnFRwWL/KEhWvkz6yw=,tag:gFyOTvrXPcUDnumAWEmP7Q==,type:str]",
+	"private_key_id": "ENC[AES256_GCM,data:U2t/nyG+vjYXsu4MkfNsPMHTCrxbn1nvhMdrWkwEhGFwNW322lJtRg==,iv:+MpwV8skwmY1I5OI/0FrAXJ2rj1XjzG0BpEhUjSiRQM=,tag:0bn6QU/JHJZr8qhiEtgZ7g==,type:str]",
+	"private_key": "ENC[AES256_GCM,data:0zbTMccCEJL61CSwu8LyTJkGH1OBbbVw/E8RsBCY32gsPQOALPyKn5eiV2jdyjcQ/MWlfydRiin0Upcoq42LsgU8krha3Lf/xCyby/U5GcJef2rauOlrhtdw8tz2jgOTvgD2qpBBwcoLhA8tuPxPy4HhSVKo/ydPIVIcvPu+xcHeGc9fxzIdSz3gVHP3ztRvlUb6nOhOKfOgGGTfBDQ7/BiBxqHRxf2Q8/WLUgSwYgaTpID7sr1BYUA51g5viFcwJL0Z0rgamki1Ti+Qa5vLq0xKJdUDWY0uVAFJtldN7gV63pdcsWJC2etWtqZCIqWdZjgN4AocwJXE6N8xycVql8F3Qb2CXUTTUv1sk1e0jJurt8vw+7h4a/OBf1StXWezHAO5xBg+xGwOUZl3VC2wzjOtqUenZe/FfmAB+s842B7iBRsYLoT01dzggu+/ArtnGUwjJFFkLwPq7n2Q7CyLUBeNEVgeh+EFDo4segeUmNe1j76/NZP1sE4lDmrKjADYMI9NiA/7rlLXi+27nGnvOJvE0X8gmtC4dbZPDwEQ3IR3Y1S6TCAx2zzTlH++0d9A8R66f4KVeEEdM302+wuI4VfiaeU5fw6wnODp7p/me9VvP5+J8HYHsczal1eSuHzcNFKIAIOH71kllapdM7F+DJV4u5BQ/uDft55fg7tP7VgX82fGU/nDGnIIIVAuyp4zHmN/BFfv8SKxnOhRWxHRXYpbeEVhmBNwAkV4FjykuRCtELXBxrgzBLMyHkFIFISlH7JmknIAtVzXZTfqHxlUFP0320U9g83I8YeLsSqzuuv2y+SmisHTMNZ+dKPzHqdzDzjqNw6LVpm/MpywEheH5ggdF2PkVWxzkub9axCHHQSdkdyZK4YvqWndUAGDptYGSdBYxkFiZ/gxHlrN+RwNFAePBhKc8p6NVDhI4sGE3Ndb0xai2YzHL7zZ7Ks3G2Yter5iFewRH/Bge1xHzqGIy2BPDzR+C5jv2n8F2xcwG7qTYCad6NvYfHeAUjgP9+36n9wmkz6zV32B8Ons1Pfqgf1LZjqDZpz9QTaqCnUklxNb2U7uaMD/iJv227s95NTrrp0ha5yZjTqDi2maVLddB4hSnKvZVxtRKDee//gX9MmXSaue88pftVlfARSIJGFPTjllPTHswRlnZYVF7d6s2eL3eTolHtTPxaRNDqor/gD4lCjgydDc3X1wkxyvcOeGvoK+coJSUmqvWUij1DnsQBvFQ/XnRIiuzcjrPAcPc2H5U064eDmm8PIQzSTsXzQ73ubVxxMyy8Wo1G8xT3EMmQjJNpddrPg9V69WsT7TFTCdfHbwSighP0r4QIuj2fsULT9UOiYzsgN28K2k4jUdHQCG6SwgLpawhwlIhJJbd8+XUVXoeuiXPZcUnOCQrM040sDDOEIQZufNC3a9RHXe0L8htFuPKNJpF3miv2yuOwXR6aOqLQG3kaCp9Fk2mTB1y6Gan/EE8GANvvCthqhPUrbt3Ho+0IQfDv6u+8P4ENLM3SpBQmwSFJCrH5sv+8l58QeTGsV2nmMU5LxOtezYgoMR1taEk5AYNHlKxxr24psODVp22Iqp9prj+CqFveA5MFLWrroPFSTNQADnSJ1Izx37Ur+OJhlLd7LZKV5CZRmXRtvPOUZSlNdG3S2Bbe4hd3jCEMqHXqRRRbe/nJZy1Bo/5l5q/CvAtNyc55qQR3OGGY4iosJkrZ/6pTUH0IKTnPB0/nQpBqIUZTXlzlybEx2SMi0ashpJnzwmbUW//ykhvA2davMWWvg2Bp0fPHa0Hz3rO7+1SAeGenV2ridTGQbrSKvHWUwzQzoP18xkZlbu1cdRgeqoMHqvpXG6jgaBNakBjNFmCTSdzR/F7hTX51yCS9c120luxuvBJ1eJqYeNJrAGHkDvN0SOQLDhMrpTkojBG5V7WRt8KPmgsXU1a/ALNycgiyzveK6tNoMB6DXxw7fGeuVRRWEKqSQaIi6jmP27oJn8V0/VcdDTFaibTutQzs0lnnMuwcJrVELjrIjeX8uoDKytkFQdxOrMjLC3BbjVpqBOCzTaNxoiDG1MOCnw2zO41TaACDZfYqFruZs/7BTu67OFQ/FAHbQ9dLJ6d+ZlipBi48SNOnjqtevBXNqAWzXzLAccXO8oEnSFko/dkVoX7KNEetD73VJlDU7x+Of/9UQHEWOIbBHb4oNOcZNic4iB5zsc1nA6ihdb5oX9YK0i+RKujvKrROfYtyYcZK9+YxvEfJjM1bahrAT+ISF6rSo=,iv:flmqreoxX3ivKFHNupXfOk+oAMRPhtrS1FSuVURMwKI=,tag:QggMaEZFUzGfUGxJCRygMw==,type:str]",
```

Comment:
is it a problem that I can see the text for all of these? Or are they properly encoded? :-)

Reply:
They are encoded as prefaced by the `ENC` part of the value. Compare, for example, with the key for the 2i2c cluster: https://github.com/2i2c-org/pilot-hubs/blob/master/secrets/2i2c.json I think what makes `sops` a little more clever than, say, `git-crypt` is that it encrypts the actual values, not just the whole file. So you can visually check the _structure_ is as expected without also needing to see the secret part.